sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1,3090 +1,3131 @@
-          "enabledDeviceCount": {
-            "format": "int64",
-            "type": "integer"
-          },
-          "disabledDeviceCount": {
-            "format": "int64",
-            "type": "integer"
-          }
+      }
+    },
+    "RegistryStatistics": {
+      "type": "object",
+      "properties": {
+        "totalDeviceCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The total number of devices registered for this hub."
+        },
+        "enabledDeviceCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The number of currently enabled devices."
+        },
+        "disabledDeviceCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The number of currently disabled devices."
         }
-      },
-      "ServiceStatistics": {
-        "type": "object",
-        "properties": {
-          "connectedDeviceCount": {
-            "format": "int64",
-            "type": "integer"
-          }
+      }
+    },
+    "ServiceStatistics": {
+      "type": "object",
+      "properties": {
+        "connectedDeviceCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The number of currently connected devices."
         }
-      },
-      "Device": {
-        "type": "object",
-        "properties": {
-          "deviceId": {
-            "type": "string"
-          },
-          "generationId": {
-            "type": "string"
-          },
-          "etag": {
-            "type": "string"
-          },
-          "connectionState": {
-            "enum": [
-              "Disconnected",
-              "Connected"
-            ],
-            "type": "string"
-          },
-          "status": {
-            "enum": [
-              "enabled",
-              "disabled"
-            ],
-            "type": "string"
-          },
-          "statusReason": {
-            "type": "string"
-          },
-          "connectionStateUpdatedTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "statusUpdatedTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "lastActivityTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "cloudToDeviceMessageCount": {
-            "format": "int32",
-            "type": "integer"
-          },
-          "authentication": {
-            "$ref": "#/definitions/AuthenticationMechanism"
-          },
-          "capabilities": {
-            "$ref": "#/definitions/DeviceCapabilities"
-          },
-          "deviceScope": {
+      }
+    },
+    "Device": {
+      "type": "object",
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "The unique identifier of this device."
+        },
+        "generationId": {
+          "type": "string",
```

Comment:
I am assuming that this and `deviceId` above will be set as `readonly`, correct?

Reply:
Yes, in a separate PR